core/src/test/scala/integration/kafka/api/AuthorizerIntegrationTest.scala

```diff
@@ -1290,7 +1314,7 @@ class AuthorizerIntegrationTest extends BaseRequestTest {
 
   private def addAndVerifyAcls(acls: Set[Acl], resource: Resource) = {
     servers.head.apis.authorizer.get.addAcls(acls, resource)
-    TestUtils.waitAndVerifyAcls(servers.head.apis.authorizer.get.getAcls(resource) ++ acls, servers.head.apis.authorizer.get, resource)
+    TestUtils.waitAndVerifyAcls(servers.head.apis.authorizer.get.getAcls(resource), servers.head.apis.authorizer.get, resource)
```

Comment:
Why was this change required? It looks like we don't wait and verify the new `acls` any more?

Reply:
@rajinisivaram it looked unnecessary : 
`servers.head.apis.authorizer.get.getAcls(resource)` already returns the `acls` that have been added to the authorizer `servers.head.apis.authorizer.get.addAcls(acls, resource)`

Reply:
was the intent to verify that addAcls does actually add them ?

Reply:
I would have thought so. Without `++ acls`, `TestUtils.waitAndVerifyAcls` doesn't really wait or verify anything.

Reply:
Oh well, it looks the test is written expecting `addAcls` to be asynchronous. 
It doesn't seem to be in the SimpleAclAuthorizer but I have no problem reverting that line.